scripts/clean_benchmarks.py

```diff
@@ -16,7 +16,7 @@ def main():
         if benchmark.is_dir():
             print(f"Cleaning {benchmark}")
             for path in benchmark.iterdir():
-                if path.name == "main_prompt":
+                if path.name in ["prompt", "main_prompt"]:
```

Comment:
I'm guessing you left the `main_prompt` for some compatibility although you renamed all prompts?

Reply:
I'd suggest `if "prompt" in path.name.lower():` for future compatibility.